src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/DefaultRazorSemanticTokenInfoService.cs

```diff
@@ -143,27 +266,47 @@ public SyntaxResult(SyntaxNode node, SyntaxKind kind, RazorCodeDocument razorCod
 
             public SyntaxKind Kind { get; set; }
 
+            public bool Transmited { get; set; }
+
+            public override bool Equals(object obj)
+            {
+                var other = obj as SyntaxResult;
+
+                var result = other.Kind == Kind && other.Range.Equals(Range);
+                return result;
+            }
+
+            public override int GetHashCode()
+            {
+                var hashCode = 352033288;
```

Comment:
Not sure I follow where these numbers are sourced. Why not use the HashCodeCombiner?

Reply:
I thought that it wasn't available because Intellisense wouldn't offer it, but apparently it's there and I just needed to include the proper `using`.